pkg/format/README.md

```diff
@@ -0,0 +1,31 @@
+format
+======
+
+Create partition table and format drives for use with LinuxKit
+
+## Usage
+
+This packages supports two modes of use:
+
+
+### Autoformat
+
+```
+onboot:
+  name: format
+  image: linuxkit/format:${hash}
+```
+
+In this mode of operation, any disks that are found that do not have a valid partition table have a table created and one linux partition created that will fill the entire disk
+
+### Manual format
+
+```
+onboot:
+  name: format
+  image: linuxkit/format:${hash}
+  command: ["/usr/bin/format", "/dev/vda", "/dev/vdb" ]
```

Comment:
We should not support multiple formats in the same command. We will need additional parameters, eg filesystem type.